src/components/TeachingBubble/TeachingBubble.Props.ts

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import { TeachingBubble } from './TeachingBubble';
+
+/**
+ * Checkbox class interface.
+ */
+export interface ITeachingBubble {
+
+}
+
+/**
+ * TeachingBubble component props.
+ */
+export interface ITeachingBubbleProps extends React.Props<TeachingBubble> {
+  /**
+   * The background color of the TeachingBubble
+   * @default white
+   */
+  backgroundColor?: TeachingBubbleBGColor;
+
+  /**
+   * A title for the Teaching Bubble.
+   */
+  title?: string;
+
+  /**
+   * An Image for the Teaching Bubble.
+   */
+  img?: string;
```

Comment:
I think this is a bit confusing and also maybe limited. Maybe this should be imageProps?: IImageProps. There are potentially other things about the image that need to be set (sizing, scaling, etc).

Alternatively... imgSrc is a better name if you absolutely don't want these to be configured.